test/test_convert.py

```diff
@@ -1,191 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
I am not sure why this show up as deleted since it was _moved_ to `test/parser/test_convert.py` above.

Reply:
stupid question: did you move it with `mv` or with `git mv`?

Reply:
Should be fixed now.